core/App/ehsm_napi.h

```diff
@@ -82,20 +93,41 @@ typedef struct {
         return StringToChar(retString);
 	};
  
-  static char* readData(char* jsonChar, std::string key){
+    static char* readData_string(char* jsonChar, std::string key){
         std::string retVal;
         std::string jsonString = jsonChar;
         key = "\"" + key + "\"";
 
-        int startIndex = jsonString.find(key) + key.size() + 1;
-        std::string subStr = jsonString.substr(startIndex);
+        int resultIndex = jsonString.find("\"result\"") + 8;
```

Comment:
What does 8 mean?  Could add some comments or use sizeof("xxxx")/strlen("xxx") instead?

Reply:
It will be fix.